scss/foundation.scss

```diff
@@ -6,7 +6,7 @@
  */
 
 // Dependencies
-@import '../_vendor/normalize-scss/sass/normalize';
+@import '../_vendor/normalize.css/normalize';
```

Comment:
As you include a CSS file, CSS will be generated even when Foundation mixins are not generated. Including a CSS file directly is not a good approach as it goes against the following (very recommanded) architecture:
1. Import
2. Configure
3. Execute (In Sass, generate CSS)

Before relying on `normalize-scss`, we had normalize included in our own mixin. This is a perfect solution but if there nothing better to get normalize updated we'll go for it.

Reply:
You can import css files in Scss without any issues. It will not generate extra files. So far normalize-scss is far behind normalize and can not be directly compared.

Reply:
I am not talking about _extra file generation_ but _systematic code generation_. This means that if I do in my code:

```scss
import 'foundation-sites`;
```

Without running `@include foundation-everything`, I'll still have normalize code generated.

Reply:
Ah, I guess I'll do some research to find a way to wrap this in a mixin. Probably by using some etract like plugin to store the content for reuse.